app/decorators/has_hint_option.rb

```diff
@@ -0,0 +1,15 @@
+class HasHintOption < SimpleDelegator
```

Comment:
This file is to tell people using the ingest form, which fields are CV's. @terrellt Do you think we should keep this in or wait until later to implement this feature?